source/simulator/core.cc

```diff
@@ -1851,6 +1857,9 @@ namespace aspect
   Simulator<dim>::
   solve_timestep ()
   {
+    // set adjoint problem to false as default
+    adjoint_problem = false;
+
```

Comment:
not longer needed?

Reply:
This is still used in postprocessing, namely in global statistics and in visualization. I need to adjoint these files first before removing that.